dewy/frontend/src/Collection.tsx

```diff
@@ -24,7 +24,7 @@ const ListActions = () => (
 export const CollectionList = () => (
     <List actions={<ListActions/>} >
         <Datagrid>
-            <TextField source="name" />
+            <TextField source="id" />
```

Comment:
The field is still `name`, here and below

Reply:
Ah, this was originally based on your PR. Not sure where that change came from.